fixtures/mock-oauth2-provider/package.json

```diff
@@ -0,0 +1,58 @@
+{
+  "name": "@loopback/mock-oauth2-provider",
+  "version": "0.0.1",
+  "description": "",
+  "private": true,
+  "engines": {
+    "node": ">=10"
+  },
+  "scripts": {
+    "build": "lb-tsc",
+    "clean": "lb-clean loopback-mock-oauth2-provider*.tgz dist *.tsbuildinfo package",
+    "pretest": "npm run build",
+    "test": "npm run mocha",
+    "mocha": "lb-mocha \"dist/__tests__/**/*.js\"",
+    "verify": "npm pack && tar xf loopback-mock-oauth2-provider*.tgz && tree package && npm run clean"
+  },
+  "author": "IBM Corp.",
+  "copyright.owner": "IBM Corp.",
+  "license": "MIT",
+  "files": [
+    "README.md",
+    "index.js",
+    "index.d.ts",
+    "dist",
+    "src",
+    "!*/__tests__"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/strongloop/loopback-next.git",
+    "directory": "fixtures/mock-oauth2-provider"
+  },
+  "dependencies": {
+    "passport": "^0.4.1",
+    "tslib": "^1.11.2",
+    "util-promisifyall": "^1.0.6"
+  },
+  "devDependencies": {
+    "@loopback/build": "^5.3.1",
+    "@loopback/eslint-config": "^6.0.6",
+    "tslib": "^1.11.2",
+    "@types/jsonwebtoken": "^8.3.9",
+    "@types/lodash": "^4.14.150",
+    "@types/node": "^10.17.21",
+    "@types/passport": "^1.0.3",
+    "@types/passport-http": "^0.3.8",
+    "@types/passport-oauth2": "^1.4.9",
+    "@types/qs": "^6.9.2",
+    "axios": "^0.19.2",
+    "body-parser": "^1.19.0",
+    "express": "^4.17.1",
+    "form-data": "^3.0.0",
+    "jsonwebtoken": "^8.5.1",
+    "lodash": "^4.17.15",
+    "passport-http": "^0.3.0",
+    "passport-oauth2": "^1.5.0"
```

Comment:
I believe some of the `devDependencies` should be moved into `dependencies` if they are used in the code of the mock app (not tests).

Reply:
agreed